src/core/Tabs/index.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import useStyles from './styles';
+import { TabBaseProps } from './base';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import AppBar from '@material-ui/core/AppBar';
+import Typography from '@material-ui/core/Typography';
+
+interface TabPanelProps {
+  children?: string;
+  index: any;
+  value: any;
+}
+
+const TabPanel: React.FC<TabPanelProps> = ({
+  children,
+  index,
+  value,
+  ...other
+}) => {
+  const classes = useStyles();
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <div className={classes.tabPanel}>
+          <Typography>{children}</Typography>
+        </div>
+      )}
+    </div>
+  );
+};
+
+const a11yProps = (index: number) => {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+};
+
+interface SiampleTabsProps extends TabBaseProps {
+  label: string[];
+  content: any[];
+}
+
+const SimpleTabs: React.FC<SiampleTabsProps> = ({ label, content }) => {
+  const classes = useStyles();
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event: any, newValue: number) => {
+    setValue(newValue);
+  };
+
+  return (
+    <div>
+      <AppBar position="static" className={classes.panel}>
```

Comment:
TabPanel, Tabs and AppBar are separate components...
see Mui API docs to make only TabPanel and Tab component

Reply:
@arkajyotiMukherjee 
These these component must be separated in the Tabs or out of it?

Reply:
TabPanel is one component and Tab will be another @saten16